feature-utils/poly-look/src/css/design-tokens.css

```diff
@@ -1,4 +1,5 @@
 :root {
   /* grid distance unit*/
   --gdu: 4px;
+  --line-height: 120%;
```

Comment:
I don't think thats a design token right? Rather a base style -> Maybe in the theme or utils?

Reply:
Yeah, I wasn't sure about this one. I've moved it from one file to another a couple of times